Formula/amber.rb

```diff
@@ -0,0 +1,18 @@
+class Amber < Formula
+  desc "CLI client for generating, scaffolding Amber web applications"
+  homepage "https://www.amberframework.org"
+  url "https://github.com/amberframework/amber/archive/v0.5.0.tar.gz"
+  sha256 "dfcf79a51a5ed6f1693cba0fc4761b4cd4cb7472f35df23182e68e4f4fa29e96"
+
+  depends_on "crystal-lang"
+
+  def install
+    system "shards", "install"
+    system "make", "install", "PREFIX=#{prefix}"
+    bin.install "bin/amber"
```

Comment:
This seems a bit odd given the `make install` step above?

Reply:
We want to move this to the users bin directory, if not done this way brew will not see the formula linked properly

Reply:
Sure, but `prefix` used above means the Cellar not the `buildpath` so why isn't `make install` obeying that rather than installing it into the `buildpath` anyway?